src/server/routes/auth-routes.js

```diff
@@ -1,78 +1,112 @@
 import express from 'express';
+import models from '../models';
+import crypto from 'crypto';
+import session from 'express-session';
+import jwt from 'jsonwebtoken';
+import config from '../config/testing';
+
 const routes = express.Router();
+const secret = config.secret; //TODO: Store secret elsewhere and change it!!
+const accessToken = config.accessToken;
 
-import models from "../models";
+// **************************** Login *******************************
 
-var passport = require('passport');
-var LocalStrategy = require('passport-local').Strategy;
-var session = require('express-session');
+var login = (user, res) => {
+  var token = jwt.sign(user, secret, {
+    expiresInMinutes: 1440 // expires in 24 hours
+  });
 
-routes.use(session({ secret: 'superSeCret' }));
-routes.use(passport.initialize());
-routes.use(passport.session());
+  user.updateAttributes({
+    last_login: new Date()
+  });
 
-// **************************** Login *******************************
+  res.cookie(accessToken, token, { maxAge: 86400000, httpOnly: true }); //might need more security to defend against CSRF:
+  //see : https://stormpath.com/blog/where-to-store-your-jwts-cookies-vs-html5-web-storage/
 
-passport.serializeUser(function(user, done) {
-  done(null, user.id);
-});
+  return res.json({
+    success: true
+  });
+};
+
+var currentUser = req => {
+  if(!(accessToken in req.cookies)) {
+    return null;
+  }
+
+  var token = req.cookies[accessToken];
 
-passport.deserializeUser(function(id, done) {
-  models.User.find(id).then(function (user) {
-    done(null, user);
+  jwt.verify(token, secret, (err, decoded) => {      
+    if (!err) { //if logged in already
+      return decoded;
```

Comment:
I dont think this will work the way you expect it to. Since we're inside a callback, returning a value won't return the value from the "parent" function. So `const x = currentUser();` will always be either `null` (if the token is invalid) or `undefined` (if the token is valid - but still undefined since we don't return anything).